### PR TITLE
stuff

### PR DESCRIPTION
**Pull request by simon-andrews**
ayy lmao

### CODE REVIEW

File: src/org/robockets/steamworks/OI.java [hunk 1]
```diff
@@ -84,27 +87,29 @@ public OI() {
...
+        //lifter1.whileHeld(new MakeExtraSpace());
+        lifter1.whileHeld(new MoveConveyor(RelativeDirection.YAxis.FORWARD));
+        lifterMan1.whileHeld(new MoveConveyor(RelativeDirection.YAxis.FORWARD));
+        lifterMan2.whileHeld(new MoveConveyor(RelativeDirection.YAxis.BACKWARD));
+		
+        climber1.whileHeld(new Climb(1));
+        //climber2.whileHeld(new Climb(1));  
+        /*
+        shooterMan1.whenPressed(new KillEverything()); */
+        //misc2.whenPressed(new Climb(1));*/
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/OI.java:113]: Why are these still commented out?

File: src/org/robockets/steamworks/Robot.java [hunk 2]
```diff
@@ -114,14 +111,14 @@ public void robotInit() {
 
 		SmartDashboard.putData(new MaxFillElevator());
 
-		RobotMap.gyro.calibrate();
+		//RobotMap.gyro.calibrate();
 
 		SmartDashboard.putNumber("GyroP", drivetrain.gyroPID.getP());
 		SmartDashboard.putNumber("GyroI", drivetrain.gyroPID.getI());
 		SmartDashboard.putNumber("GyroD", drivetrain.gyroPID.getD());
 		SmartDashboard.putNumber("GyroSetpoint", drivetrain.gyroPID.getSetpoint());
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/Robot.java:134]: DUMPER

File: src/org/robockets/steamworks/Robot.java [hunk 3]
```diff
@@ -131,13 +128,15 @@ public void robotInit() {
 		Webcam.getInstance().startThread();
 
 		autoTest = new AutoTest();
-		autonomousChooser = new SendableChooser<>();
+		autonomousChooser = new SendableChooser<Command>();
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/Robot.java]: This is redundant

File: src/org/robockets/steamworks/RobotMap.java [hunk 4]
```diff
@@ -55,7 +55,7 @@
      * Speed controller for the roller at the bottom of the robot that sucks
      * fuel balls in.
      */
-    public static Victor ballIntakeRollerSpeedController = new Victor(7); //TODO: get real port
+    public static Victor ballIntakeRollerSpeedController = new Victor(6); //TODO: get real port
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/RobotMap.java]: // VICTOR 7 would go here

**Review by UnknownSilicon — APPROVED** [review]
It's good but there are some misc fixes needed

File: src/org/robockets/steamworks/OI.java (hunk 1 above)
Comment (simon-andrews) [reply src/org/robockets/steamworks/OI.java:113]: Launchpad issues. We'll uncomment them when we test them """today""".

File: src/org/robockets/steamworks/Robot.java (hunk 2 above)
Comment (simon-andrews) [reply src/org/robockets/steamworks/Robot.java:134]: On it.

Comment (simon-andrews) [reply src/org/robockets/steamworks/Robot.java:134]: Un-on it.

**Pull request merged**